app/Models/Foundation/Main/Audit/AuditLog.php

```diff
@@ -23,6 +23,7 @@
  * @ORM\DiscriminatorMap({
  *     "SummitAuditLog" = "SummitAuditLog",
  *     "SummitEventAuditLog" = "SummitEventAuditLog",
+ *     "SummitAttendeeBadgeAuditLog" = "SummitAttendeeBadgeAuditLog"
```

Comment:
you need to add a migration to update the Enumerate

Reply:
done